accounts-db/src/accounts_db.rs

```diff
@@ -7578,15 +7577,9 @@ impl AccountsDb {
             .thread_pool
             .install(|| AccountsDeltaHash(AccountsHasher::accumulate_account_hashes(hashes)));
         accumulate.stop();
-        let mut uncleaned_time = Measure::start("uncleaned_index");
-        self.uncleaned_pubkeys.insert(slot, dirty_keys);
```

Comment:
If we are flushing slots 1 and 2 at some later point and a pubkey is in both slots 1 and 2, then we'll only `should_flush`=`true` for the pubkey in the higher slot (2) inside of flush. The current code here will add the pubkey to BOTH slots 1 and 2. Does that matter? I'm not sure. We definitely need to `clean` the index entry for the pubkey in slot 1. Maybe that happens some other way in flush?